rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2493,6 +2493,12 @@ def test_upgrade_db_43_to_44(user_data_dir, messages_aggregator):
             (bad_address, None, 'yabir.eth'),
         }
 
+    with db_v43.conn.write_ctx() as cursor:
+        cursor.execute(
```

Comment:
nitpick: Name write cursors accurately

```suggestion
    with db_v43.conn.write_ctx() as write_cursor:
        write_cursor.execute(
```